### DISCUSSION TITLE
账号显示因为NSFW被封禁

### DISCUSSION THREAD

**Discussion by Tatsumi-Ryuu** [BurnHair/docs#136] [技术支持]
用户id：3198
沉浸式翻译开了个自动模式，之前没上网页，就没看到公告，充了钱还没怎么用，还有机会解封么

**Comment by BennyThink** [comment]
仅可以解封一次，下不为例所以要注意了哦

**Reply by Tatsumi-Ryuu** [reply]
> 仅可以解封一次，下不为例所以要注意了哦

好的好的，我已经把沉浸式翻译关了，后面应该不会了
